### DISCUSSION TITLE
Upgrade minor Argocd versions

### DISCUSSION THREAD

**Discussion by guy-frontegg** [argoproj/argo-cd#8200] [Q&A]
Hello All,

We have multiple clusters with argocd. One was installed using kubectl and it's version 2.1.0 and the others were installed via helm and the version is 2.1.16.

We would like to upgrade all servers to 2.2.2 and we are a little concern with the process. AS we understand the CRD were changed.

What are the best practice to upgrade argocd in production environment without the risk of deleting or damaging "live" applications.

**Comment by ishitasequeira** [comment]
You can follow https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/overview/ for upgrading ArgoCD.
1) Check the upgrade notes for the particular version you are upgrading to
2) Do `kubectl apply` on the new version
